docs/installing-solidity.rst

```diff
@@ -216,6 +216,21 @@ Alternatively, you can build for Windows on the command-line, like so:
 
     cmake --build . --config RelWithDebInfo
 
+The version string in detail
+============================
+
+The Solidity version string contains four parts:
+- the version number
+- pre-release tag, usually set to ``develop.YYYY.MM.DD`` or ``nightly.YYYY.MM.DD``
+- commit in the format of ``commit.GITHASH``
```

Comment:
Can you perhaps also explain that the semver pre-release part consists of this pre-release tag (naturally) while the semver build info consists of commit information plus platform?